ci/cirrus-templates/freebsd.yaml

```diff
@@ -0,0 +1,24 @@
+# This is ci/actions-templates/linux-builds-template.yaml
+# Do not edit this file in .cirrus.yml
+
+task:
+  name: FreeBSD
+  freebsd_instance:
+    image: freebsd-13-0-release-amd64
+  setup_script: |
+    pkg install -y git gmake bash
+    echo "========="
+    echo "create non-root user and log into it"
+    pw group add -n tester
+    pw user add -n tester -g tester -s `which bash`
+    pw user mod tester -d `pwd`
+    chown -R tester .
+    sudo -u tester bash ci/cirrus-templates/script.bash
+  upload_to_s3:
+    only_if: $CIRRUS_BRANCH == "stable"
+    script:
+      - pkg install -y py36-pip bash
+      - pip-3.6 install -U setuptools
+      - pip-3.6 install awscli
+      - bash ci/prepare-deploy.bash
+      - aws s3 cp --recursive deploy/ s3://dev-static-rust-lang-org/rustup/
```

Comment:
Do we need to set AWS_DEFAULT_REGION here?

Reply:
No idea. I couldn't test this part so if something needs to be added, please tell me. :)

Reply:
I'm tempted to say "remove this" and we'll stick with stable releases from the cross-built binary for now.  Otherwise we need to be sure about this *and* disable the freebsd cross builds.

Reply:
Done! I have put the removal in its own commit in case we want to have the S3 upload later on so it's easy to revert.